### DISCUSSION TITLE
v1.0.15

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense-companion#934] [Announcements]
## What's Changed
Add 3d view (#927)

## Chores
* Add some bounds checking by @DTTerastar in https://github.com/ESPresense/ESPresense-companion/pull/876
* Bump eslint from 9.14.0 to 9.15.0 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/875
* Bump tslib from 2.8.0 to 2.8.1 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/874
* Bump @typescript-eslint/eslint-plugin from 8.14.0 to 8.15.0 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/872
* Bump @types/node from 22.9.0 to 22.9.1 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/871
* Bump Swashbuckle.AspNetCore from 7.0.0 to 7.1.0 in /src by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/887
* Update eslint by @DTTerastar in https://github.com/ESPresense/ESPresense-companion/pull/926
* Bump @sveltejs/kit from 2.8.1 to 2.15.1 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/924
* Bump YamlDotNet from 16.2.0 to 16.3.0 in /src by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/923
* Bump prettier-plugin-svelte from 3.2.8 to 3.3.2 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/882
* Bump svelte-check from 4.0.9 to 4.1.1 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/897
* Bump Swashbuckle.AspNetCore from 7.1.0 to 7.2.0 in /src by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/905
* Bump typescript from 5.6.3 to 5.7.2 in /src/ui by @dependabot in https://github.com/ESPresense/ESPresense-companion/pull/883

**Full Changelog**: https://github.com/ESPresense/ESPresense-companion/compare/v1.0.14...v1.0.15

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense-companion/releases/tag/v1.0.15'>v1.0.15</a>.</em>

**Comment by leccelecce** [comment]
Might be nice to give a little credit for the 3D view to https://github.com/hdurdle/espc3d and https://github.com/leccelecce/espc3d? I even posted the link to my repo on Discord 4 months ago...
